File: Modulo2-ObjectiveC-Lorena/Exercicio10.m
//
//  main.m
//  Exercicio10-C
//
//  Created by Lorena Rodrigues Bruno on 06/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        int numero, i;
        printf("Digite um número:");
        scanf("%i", &numero);
        
        for(i=2; i< numero; i++) {
            if(numero % i == 0) {
                printf("O número %i não é primo\n", numero);
                break;
            }
            if(i == numero-1) {
                printf("O número %i é primo\n", numero);
            }
        }
    }
    return 0;
}

File: Modulo2-Lorena/exercicio10.swift
//
//  main.swift
//  Exercicio10
//
//  Created by Lorena Rodrigues Bruno on 24/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

print("Digite um número:")
var entrada : String! = readLine()
var numero : Int = Int(entrada)!
var contador = 1

for contador in 2...numero-1 {
    if numero%contador == 0 {
        print("O número \(numero) não é primo!")
        break
    }
    if contador == numero-1 {
        print("O número \(numero) é primo!")
    }
}




